out/WIPL.asm

```diff
@@ -128,18 +120,21 @@ BackToPrompt:
     ; According to known behavior, thus jumps back to WIPL prompt, effectively
     ; restarting it
 01bb:    04           ei
-01bc:    47 4c        unk7 GH, RH
-01be:    00           HALT
-01bf:    ff           st BX, (HX)
-01c0:    02           sf
-01c1:    65 47        ld RT, unknown_indexed
-01c3:    9c           ld AX, (EX)
-01c4:    09           ret
-
-Entry_0x1c5:
-01c5:    a0 03        st AL, #0x03
-01c7:    65 79 05     ld RT, +0x5(DX)+
-01ca:    5a           and! BX, AX
+    ; We don't currently understand what 47 instruction really does, but this
+    ; is the only reference to 0x0265, where a "skip code check" flag is stored.
+    ; And we also know that in pristine WIPL, obtained from disk dump, the respective
+    ; location contains zero; while in this one, which has been extracted by dumping
+    ; from the TOS after it hit the "Stop" loop and the machine has been restarted,
+    ; we see a 0xff. So, apparently, this sequence stores an 0xFF at 0x0265
+01bc:    47 4c        unk7 GH, RH	 ; fill
+01be:    00           (0x0)	 ; 1 byte
```

Comment:
Using the fill instruction to write a single byte is weird. Are you sure this isn't wrapping around to a 256 byte fill?

The next instruction starts it's fill at 0x365, which is 256 bytes after this fill.